### ISSUE TITLE
Updating variables is slow

### ISSUE THREAD

**Issue by DillonJ** [spine-tools/SpineOpt.jl#558]
I have a model with 500 nodes and 700 branches. It's performs pretty well considering but it takes 30 seconds per window at "updating variables". Feels like we should be able to do better?

Any thoughts @manuelma

**Comment by manuelma** [comment]
contingency_is_binding?

**Comment by DillonJ** [comment]
No, with LODF constraint active

**Comment by manuelma** [comment]
Yeah, sorry, contingency is binding has nothing to do with updating variables.

Updating variables at the moment is fixing the history after rolling (to the results of the last solve) *and* also fixing values and restoring lower and upper bounds.

I think I can make it faster, I have some work on a branch but need to test it before merging into master.

**Comment by manuelma** [comment]
That branch is already merged. I was testing it with a model where it took 4 seconds per window - but it was 5 seconds before merging, so either @DillonJ 's model was taking 30 seconds because of other issues of because it's just larger...
